examples/walkthrough/README.md

```diff
@@ -0,0 +1,147 @@
+# Walkthrough
+
+## Setup
+
+### [Install Dependencies](https://github.com/smartcontractkit/chainlink#install)
+
+For this walkthrough, you will need:
+
+- [Go](https://golang.org/doc/install#install)
+- [dep](https://github.com/golang/dep#installation)
+- [Node](https://nodejs.org/en/download/package-manager/)
+- [Yarn](https://yarnpkg.com/lang/en/docs/install/#mac-stable)
+- [Docker](https://www.docker.com/get-started)
+- [direnv](https://direnv.net/) (optional)
+
+If you're on a mac with [Homebrew](https://brew.sh/), you can run:
+
+`brew install go dep node yarn docker direnv`
+
+### Install Chainlink
+
+
+```bash
+go get -d github.com/smartcontractkit/chainlink
+cd $GOPATH/src/github.com/smartcontractkit/chainlink
+make install
```

Comment:
Optional nitpick:

This has been changed to be `make bootstrap` because of gencodec: 

```
go get -u github.com/smartcontractkit/gencodec
```

It is unlikely that they will need it, but thought it was worth a mention.